src/screens/menu_main.py

```diff
@@ -49,14 +50,14 @@ def __init__(
     def reset_fields(self) -> None:
         """Reset all input fields and hide them."""
         self.input_active = False
-        self.initials_active = False
+        self.players_name_active = False
         self.input_text = ""
-        self.initials_text = ""
+        self.players_name_text = ""
         self.play_button_enabled = False
 
-    def validate_initials(self, initials: str) -> bool:
-        """Validate if initials are exactly 2 characters and only letters."""
-        return len(initials) == MAX_INITIALS_LEN and initials.isalpha()
+    def validate_players_name(self, players_name: str) -> bool:
+        """Validate if players_name are exactly 2 characters and only letters."""
```

Comment:
I think this documentation line is wrong:
The function actually """Validate if `player_name` is shorter than `MAX_INITIALS_LEN` and alphanumeric."""

Reply:
Fixed in PR repository.

Reply:
Tested locally and everything works fine.